utils/post_utils.test.jsx

```diff
@@ -725,12 +728,13 @@ describe('PostUtils.createAriaLabelForPost', () => {
         const isFlagged = true;
 
         const ariaLabel = PostUtils.createAriaLabelForPost(testPost, author, isFlagged, reactions, intl, emojiMap);
-        assert.ok(ariaLabel.indexOf('smile emoji'));
-        assert.ok(ariaLabel.indexOf('+1 emoji'));
-        assert.ok(ariaLabel.indexOf('non-potable water emoji'));
-        assert.ok(ariaLabel.indexOf(':space emoji:'));
-        assert.ok(ariaLabel.indexOf(':not_an_emoji:'));
+        expect(ariaLabel.indexOf('smile emoji')).toBeTruthy();
+        expect(ariaLabel.indexOf('+1 emoji')).toBeTruthy();
+        expect(ariaLabel.indexOf('non-potable water emoji')).toBeTruthy();
+        expect(ariaLabel.indexOf(':space emoji:')).toBeTruthy();
+        expect(ariaLabel.indexOf(':not_an_emoji:')).toBeTruthy();
```

Comment:
I know this code was already here but checking that `indexOf`'s return value is truthy seems off to me. Isn't `0` the only value that's falsy? `-1` is treated as truthy.

Reply:
That's a good catch. I'll fix that.